docs/BUILD.md

```diff
@@ -364,22 +364,28 @@ $ make proto
 Each package can be built independently via:
 
 ```
-linuxkit pkg build <directory>
+make pkg/<name>
 ```
 
 For example, to build `guacd`:
 
 ```
-linuxkit pkg build pkg/guacd
+make pkg/guacd
 ```
 
-Or from within the `pkg/guacd` directory:
+To build all of the dependent packages:
 
 ```
-linuxkit pkg build .
+make pkgs
 ```
 
-To simplify and collate building, you can run `make build` in the `pkg/` directory, or just `make -C pkg/ build`. This will build all of the dependent packages.
+In some cases, the `pkg/<name>` rule may rebuild more than intended, as they
+specify required dependencies.  In this case, it may be more efficient to use
+`make eve-<name>`, such as when testing changes in the `pillar` package:
+
+```
+make eve-pillar
+```
```

Comment:
Agreed that the line 382 with "`make build` in the `pkg/` directory" should be removed, as it is no longer correct.

But `make eve-<pkg>` is identical to `make pkg/<pkg>`. It is just a convenience definition:

```makefile
pkg/%: eve-% FORCE
        @true
```

So the whole paragraph is incorrect. Can we just change it to say something like, "for convenience, you may call `make eve-<pkg>` or `make pkg/<pkg>`?